src/rtc/mosaic_geobursts.py

```diff
@@ -0,0 +1,291 @@
+'''
+A module to mosaic Sentinel-1 geobursts from RTC workflow
+'''
+
+import os
+
+import numpy as np
+from osgeo import gdal
+
+
+def check_mosaic_eligibility(list_rtc: list, list_nlooks: list) -> bool:
+    '''
+    Check if the list of the geobursts are eligible to be mosaiced
+
+    Parameters:
+    -----------
+        list_rtc: list
+            path to the RTC geoburst
+        list_nlooks: list
+            path to the nlooks raster that corresponds to list_rtc
+
+    Returns:
+        flag_rtn: bool
```

Comment:
please don't use these names that are difficult to understand. What is `rtc`?

Reply:
As discussed, the variable names will be revised to avoid confusion.